teams.md/docs/csharp/getting-started/running-in-teams.md

```diff
@@ -6,15 +6,15 @@ sidebar_position: 3
 
 Now that your agent is running locally, let's deploy it to Microsoft Teams for testing. This guide will walk you through the process.
 
-## Teams Toolkit
+## M365 Agents Toolkit
 
-Teams Toolkit is a powerful tool that simplifies deploying and debugging Teams applications. It automates tasks like managing the Teams app manifest, configuring authentication, provisioning, and deployment. If you'd like to learn what it helps automate, check out [Teams core concepts](/teams/core-concepts).
+M365 Agents Toolkit is a powerful tool that simplifies deploying and debugging Teams applications. It automates tasks like managing the Teams app manifest, configuring authentication, provisioning, and deployment. If you'd like to learn what it helps automate, check out [Teams core concepts](/teams/core-concepts).
 
-### Install Teams Toolkit (TTK)
+### Install M365 Agents Toolkit (ATK)
```

Comment:
Please remove (ATK). John Miller asked us to not reference the toolkit as ATK in official docs scenarios, and this counts.

Reply:
Imo, if we keep it as atk in our CLI, then this is fine, because it's sort of referencing what it'll be called in our CLI. maybe we change it to ("aliased ATK in our CLI")?